packages/collector/test/tracing/sdk/allowRootExitSpans/app_with_entry.js

```diff
@@ -12,7 +12,8 @@ process.on('SIGTERM', () => {
 
 const instana = require('../../../../src')({
   tracing: {
-    allowRootExitSpan: true
+    allowRootExitSpan: true,
+    useOpentelemetry: false
```

Comment:
I saw Otel spans when running the test. I have not looked why. But disabling in general is mostly better anyway for our tests